radio/src/gui/128x64/model_logical_switches.cpp

```diff
@@ -509,6 +516,12 @@ void menuModelLogicalSwitches(event_t event)
         case LS_FIELD_V3:
           cs->v3 = CHECK_INCDEC_PARAM(event, cs->v3, v3_min, v3_max);
           break;
+        case LS_FIELD_DURATION:
```

Comment:
I don't think it's a good idea to move the case order, it's better to keep the order on the LCD